src/common/utils/storage/storage.ts

```diff
@@ -0,0 +1,19 @@
+export function getStorageItem<T>(key: string): T | undefined {
```

Comment:
If only place where the functions of this file are applied is in auth-context.ts file, then I think it might be too early to refactor this to its own module. It makes reading little bit harder imo.

Reply:
Refactored to auth-context